### ISSUE TITLE
[docs] Speedups enabled by default

### ISSUE THREAD

**Issue by floggle** [Toblerity/Shapely#661]
Shapely==1.6.4.post1

The docs state that shapely.speedups is *disabled* by default, but for the version I downloaded, they're enabled by default.
That also appears to be what the code itself is saying:

https://github.com/Toblerity/Shapely/blob/0d72c3d63802c9d16cf235bb5166b11384730226/shapely/speedups/__init__.py

```
# if cython speedups are available, use them by default
if available:
    enable()
```

**Comment by sgillies** [comment]
@floggle, right, we switched the defaults a while back and didn't update the docs.

**Comment by sgillies** [comment]
Fixed in 5e4561c.
